packages/evm/src/precompiles/14-kzg-point-evaluation.ts

```diff
@@ -21,22 +21,21 @@ export async function precompile14(opts: PrecompileInput): Promise<ExecResult> {
   const commitment = opts.data.slice(96, 144)
   const kzgProof = opts.data.slice(144, 192)
 
-  if (bufferToBigInt(z) >= BLS_MODULUS || bufferToBigInt(y) >= BLS_MODULUS) {
+  if (bytesToBigInt(z) >= BLS_MODULUS || bytesToBigInt(y) >= BLS_MODULUS) {
     return EvmErrorResult(new EvmError(ERROR.POINT_GREATER_THAN_BLS_MODULUS), opts.gasLimit)
   }
 
   if (
-    bufferToHex(Buffer.from(computeVersionedHash(commitment, version))) !==
-    bufferToHex(versionedHash)
+    bytesToHex(Buffer.from(computeVersionedHash(commitment, version))) !== bytesToHex(versionedHash)
```

Comment:
Would it work to:
- Replace Buffer.from() with toHex or the Uint8Array constructor 
- Compare the bytes directly with the equalBytes (iirc) util from ethereum-cryptography